ds4drv/uinput.py

```diff
@@ -344,10 +344,10 @@ def emit(self, report):
         self.device.syn()
 
     def emit_reset(self):
-        """Resets the device to a blank state."""
+        """Resets the device to a zero state."""
```

Comment:
Is the zero state the neutral position of the joystick inputs?

Reply:
Yes.  Previously it was one of the extremes